src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -657,6 +657,18 @@ public virtual int? SomeDacField{ get; set; }</value>
   <data name="PX1076Title" xml:space="preserve">
     <value>This code calls Acumatica internal API marked with PXInternalUseOnlyAttribute which is not intended for public use</value>
   </data>
+  <data name="PX1077Fix" xml:space="preserve">
+    <value>Change the accessibility modifier to {0}</value>
+  </data>
+  <data name="PX1077Fix_RemoveExplicitInterface" xml:space="preserve">
```

Comment:
We should not remove explicit interface implementation. 
In a scenario when event handler is an explicit interface, we should just report the event handler as an error to developers, and leave it for them to decide how to fix the error.

If graph event handler is an explicit interface implementation then it is certainly non-standard situation, and we don't have enough information to automatically correctly resolve it. We can't provide a safe recommendation here. In such case the best we can do is to leave it up to the developer.

Reply:
Done.